manifests/init.pp

```diff
@@ -55,8 +55,8 @@
   Integer[0] $hosts_queue_workers = 1,
 ) {
   class { 'katello::params':
-    candlepin_oauth_key    => $candlepin_oauth_key,
-    candlepin_oauth_secret => $candlepin_oauth_secret,
+    candlepin_oauth_key    => Sensitive($candlepin_oauth_key),
```

Comment:
My Guideline is:   
If You get a `Sensitive`, then you must pass it over as `Sensitive`, and if the receiving Module cannot deal with it, then Work should be done to improve this.  
If you already get an un-`Sensitive`, then there is no need to cast it to `Sensitive`, because the Damage already happend.  Work should be done on the sending Side to not receive this un-`Sensitive`.

Reply:
I was conservative here: I don't yet know if our installer handles it properly and only `init.pp` is exposed there. But I'll evaluate it more in depth